README.md

```diff
@@ -12,7 +12,7 @@ Prerequisites: fh-android-sdk : 3.0.+, Android Studio : 1.4.0 or newer, Android
 
 ## What is it?
 
-This application manages items in a collection that is synchronized with a remote RHMAP cloud application.  The user can create, update, and delete collection items.  Refer to `fhconfig.properties` and `ListItemsFragment.java` for the delevant pieces of code and configuraiton.
+This application manages items in a collection that is synchronized with a remote RHMAP cloud application.  The user can create, update, and delete collection items.  Refer to `app/src/main/assets/fhconfig.properties` and `app/src/main/java/redhat/com/syncsample/view/ListItemsFragment.java` for the delevant pieces of code and configuraiton.
```

Comment:
`app/src/main/java/redhat/com/syncsample/view/ListItemsFragment.java` is wrong. The correct is `app/src/main/java/com/feedhenry/sync/activities/ListOfItemsActivity.java`